### PR TITLE
Added a print stylesheet

### PR DESCRIPTION
**Pull request by pmahnke**
## Done
* Added a new [print stylesheet](http://0.0.0.0:8001/static/css/print.css)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- print any page to see that the header, nav, footer and back-to-top links are removed
- see the margins look ok
- see that external links urls are printed

### CODE REVIEW

File: static/sass/print.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+a:visited,
+a {
+  background: transparent;
+  color: $color-link;
+  font-weight: bold;
+  text-decoration: underline;
+  text-align: left;
+}
+
+
```
Comment (tbille) [inline static/sass/print.scss]: Remove line

**Comment by frankiegranato** [comment]
Can we please add fonts and colour styles as well? This is not a show-stopper though.

**Comment by pmahnke** [comment]
@frankiegranato added fonts... doing very little to change colours though

@tbille re-did the whitespace

**Comment by frankiegranato** [comment]
I checked the generated pdf. Its an improvement for sure.

**Comment by anthonydillon** [comment]
## QA 
- Can you hide the live chat thingy
- Can you look at adding a page-break to the strips

**Comment by cjohnston1158** [comment]
It looks like the TOC is still showing up when printing.

**Comment by anthonydillon** [comment]
@cjohnston1158 thanks.

This branch needs to be rebased on master. So the TOC are nested in a nav element.

**Comment by pmahnke** [comment]
@anthonydillon 

Hid more patterns from printing
* not printing
  * #mobile_invitation_container,
  * .p-top,
  * .u-hide--small,
  * .p-heading-icon__img,
  * .p-contextual-footer
* max sizing logos
* updated the class `p-hidden` and `u-hidden` to `u-hide`
* make p-strips avoid page breaking

**Comment by anthonydillon** [comment]
@pmahnke did you merge master in or rebase?

**Comment by anthonydillon** [comment]
@pmahnke did you merge master in or rebase?

**Comment by anthonydillon** [comment]
@pmahnke I still see the live chat widget.
[cloud-openstack.pdf](https://github.com/canonical-websites/www.ubuntu.com/files/1555473/cloud-openstack.pdf)

**Comment by anthonydillon** [comment]
LGTM :+1:

**Comment by anthonydillon** [comment]
Might want to try and fix up that merge first.

**Pull request merged**